creator-node/src/ffmpeg.js

```diff
@@ -63,24 +64,27 @@ function segmentFile(fileDir, fileName, { logContext }) {
     proc.stderr.on('data', (data) => (stderr += data.toString()))
 
     proc.on('close', (code) => {
-      if (code === 0) {
-        const segmentFileNames = fs.readdirSync(fileDir + '/segments')
-        const segmentFilePaths = segmentFileNames.map((filename) =>
-          path.resolve(fileDir, 'segments', filename)
-        )
+      async function asyncFn() {
+        if (code === 0) {
+          const segmentFileNames = await fs.readdir(fileDir + '/segments')
```

Comment:
i don't think there's a readdir promise version, it's just a passthrough of the fs function https://github.com/jprichardson/node-fs-extra#async

Reply:
confirmed specifically `fs-extra.readdir` returns promise
```
const fs = require('fs-extra')

// returns Promise { <pending> }
console.log(fs.readdir('./dir'))
```